contents/blog/best-open-source-ab-testing-tools.md

```diff
@@ -0,0 +1,165 @@
+
+[PostHog’s Experimentation suite](https://posthog.com/docs/user-guides/experimentation) allows you to flexibly test your hypotheses with up to three test variants. A control group is also assigned and PostHog will automatically estimate the number of users exposed, and we use Bayesian analysis to calculate the results and ensure they're significant.
+
+### Who is PostHog for?
+
+PostHog is a great choice for any team that wants a complete view how people use their product. Not only is the testing functionality comprehensive, it’s tightly integrated with a complete suite of product analytics tools – you don’t need any external tools. You can also [self-host PostHog](/signup/self-host), so data never leaves your infrastructure.
+
+### Strengths
+
+- Full product analytics suite
+- Easy to get set up – no SQL necessary
+- Ability to select segments for experiments
+- Feature flags for incremental roll outs
+- Event pipelines for integration with data warehouses
+- Traffic visualization with user path analysis
+- Self-hosted and cloud-hosted options available
+
+### How much does it cost?
+
+We’re confident PostHog is the number one choice here, but if you want to test our theory you can [try our self-serve plans for free](/pricing). 
+
+If you want to self-host, you can use Experimentation with our Scale plan, but you can also use PostHog Cloud if you prefer a fully-managed option with no infrastructure to deploy. 
+
+Both PostHog Scale and Cloud are [free up to 1 million events per month](/pricing).
+
+### Open source license
+
+PostHog Open Source is available under an MIT license. You can [see further details on GitHub]. Experimentation isn't part of our open source release as it requires a Scale license, but Scale is free to use up to 1 million events per month and you can set event limits to ensure you don’t go over. Read our docs for more on [how to self-host PostHog](https://posthog.com/docs/self-host).
+
+<ArrayCTA />
+
+**Further reading:**
+- [The best open source Hotjar and FullStory alternatives](/blog/best-open-source-session-replay-tools)
+- [What we've learned about running effective A/B tests](/blog/experiments)
+- [The differences between PostHog and Amplitude](/blog/posthog-vs-amplitude)
+
+## 2. [GrowthBook](https://www.growthbook.io/)
+
+![GrowthBook - best open source ab testing tools](../images/blog/open-source-testing-tools/growthbook.png)
+
+GrowthBook is an open-source platform for feature flags and A/B tests which helps teams to deploy code efficiently and analyze experiments. Built by engineers who wanted better adoption insights into their releases, it is a modular solution that promotes feature flagging as an essential step in the development process and can be used as a full-stack platform, a plugin feature flagging tool or an analysis engine.
```

Comment:
Reads weirdly - 'Built by engineers who wanted better insights into adoption of their releases, etc.' maybe?